apps/prairielearn/src/server.js

```diff
@@ -673,7 +673,7 @@ export async function initExpress() {
         res.locals.navPage = 'job_sequence';
```

Comment:
Bonus points if you can more this into the page template (assuming this is necessary - the other mount points don't have this AFAICT).

Reply:
This is actually not needed, as everywhere where the `navPage` is used in the code reached by this page, `job_sequence` does not lead to any meaningful functionality, so I removed it.